src/environments/environment.prod.ts

```diff
@@ -0,0 +1,12 @@
+import { AppConfig } from '../config/app-config.interface';
```

Comment:
Would there be any way to rename this file? Even call it `environment.production.ts`?  My concern here is that when someone upgrades from 7.1 -> 7.2, if they have an existing `src/environments/environment.prod.ts` (for their v7.1 production configs), it may be **overwritten** by this new file.

So, either we need to find a way to avoid this file overwritting any existing `environment.prod.ts` **OR** we'd have to careful document that you must copy your existing `environment.prod.ts` to a different location before upgrading.

Reply:
It can be refactored to `environment.production.ts` without any other change.

Reply:
Good foresight. Will refactor to `environment.production.ts`.

Reply:
I think documentation is the way to go. Otherwise not sure which one will be picked up if both left in place.

Reply:
@tdonohue I was not 100% correct. If we change `environment.prod.ts` to `environment.production.ts` it requires updating angular.json to explicitly express the file to replace for production build configuration. This means we can refactor and update angular.json and do not have to worry about existing `environment.prod.ts` from being used. However, git will think it is renamed and could cause merge conflicts when pull latest release.

Reply:
@wwelling : We *might* be able to remove that git issue by squashing the commits here a bit more.  As the `environment.prod.ts` didn't exist in the codebase in 7.1, Git shouldn't be aware of it _until you added it in this PR_.  So, if you squash the commit where you added it with the commit where you renamed it, then it should just appear under the name `environment.prodution.ts`...and Git would NOT think it's a rename.

At least, that's my best guess as to what Git is seeing in this PR

Reply:
Thanks. I will give that a try.

Reply:
Further investigation yields there will be no conflict. The `environmnet.produciton.ts` is a new file. Sorry to mislead on potential conflicts.